gnovm/docs/go-gno-compatibility.md

```diff
@@ -288,7 +288,7 @@ Additional native types:
 | os/exec                                     | TBD      |
 | os/signal                                   | TBD      |
 | os/user                                     | TBD      |
-| path                                        | TBD      |
+| path                                        | full     |
```

Comment:
I'd say it's `partial` if there's no `match.go` -- so either add also `match.go` or change this to partial :)